### PR TITLE
Dtspo 14838 add postgres dbs

### PR DESCRIPTION
**Pull request by cpareek**
**Before creating a pull request make sure that:**

- [ X] commit messages are meaningful and follow good commit message guidelines

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-14838

### Change description ###

This duplicate PR from - https://github.com/hmcts/sscs-tribunals-case-api/pull/3422 

Added postgres databases using chart-postgresql

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

**Review by endakelly — APPROVED** [review]

File: charts/sscs-tribunals-api/values.ccd.preview.template.yaml [hunk 1]
```diff
@@ -274,7 +259,7 @@ ccd:
       01_input.conf: |
         input  {
           jdbc {
-            jdbc_connection_string => "jdbc:postgresql://${SERVICE_NAME}-postgresql:5432/data-store?ssl=disable&stringtype=unspecified"
+            jdbc_connection_string => "jdbc:postgresql://${SERVICE_NAME}-postgresql:5432/data-store?sslmode=require&stringtype=unspecified"
```
Comment (timja) [inline charts/sscs-tribunals-api/values.ccd.preview.template.yaml:262]: this doesn't look right

Comment (cpareek) [reply charts/sscs-tribunals-api/values.ccd.preview.template.yaml:262]: logstash: has been disabled in the CCD chart  so changing here wouldn't have got tested so will change it back..

**Review by timja — COMMENTED** [review]
but logstash will need to work and read from the right database? 

i think this will need fixing too

**Comment by cpareek** [comment]
> but logstash will need to work and read from the right database?
> 
> i think this will need fixing too

My bad -  logstash is enabled from the sscs-tribunals-case-api  but it was disabled only from the chart-ccd , so sscs-tribunals-case-api' s  chart is overwriting the disabled value.

Logstash pod is running correctly..
image

**Comment by timja** [comment]
and which database is it connected to?

**Comment by timja** [comment]
logstash is full of these errors:

```
Caused by: java.net.UnknownHostException: sscs-tribunals-api-pr-3437-postgresql
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:220) ~[?:?]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[?:?]
	at java.net.Socket.connect(Socket.java:609) ~[?:?]
	at org.postgresql.core.PGStream.createSocket(PGStream.java:241) ~[postgresql.jar:42.3.3]
	at org.postgresql.core.PGStream.<init>(PGStream.java:98) ~[postgresql.jar:42.3.3]
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:109) ~[postgresql.jar:42.3.3]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:235) ~[postgresql.jar:42.3.3]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql.jar:42.3.3]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql.jar:42.3.3]
	at org.postgresql.Driver.makeConnection(Driver.java:400) ~[postgresql.jar:42.3.3]
	at org.postgresql.Driver.connect(Driver.java:259) ~[postgresql.jar:42.3.3]
	at jdk.internal.reflect.GeneratedMethodAccessor44.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
```

because it's trying to connect to a pod database and not the one that's being setup here

**Review by timja — CHANGES_REQUESTED** [review]
logstash needs fixing

File: charts/sscs-tribunals-api/values.ccd.preview.template.yaml [hunk 2]
```diff
@@ -216,7 +216,7 @@ ccd:
     
   logstash:
     image: hmctspublic.azurecr.io/imported/logstash/logstash
-    imageTag: "7.16.1"
+    imageTag: "7.16.1-probateman"
```
Comment (timja) [inline charts/sscs-tribunals-api/values.ccd.preview.template.yaml:219]: err this won't work what are you trying to do here?

**Comment by cpareek** [comment]
Have to close this PR for random error related to PR build

**Pull request closed without merging**